crates/forge/tests/data/fuzzing/tests/incorrect_args.cairo

```diff
@@ -6,6 +6,13 @@ fn correct_args(b: felt252) {
     assert(result == 2 + b, '2 + b == 2 + b');
 }
 
+#[test]
+#[fuzzer(test: 10)]
+fn incorrect_attribute_ignored(b: felt252) {
```

Comment:
Shouldn't we throw an error in case like this?

Reply:
Yeah, i'll update it as soon as #760 is merged